index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="styles/style.css" rel="stylesheet">
+</head>
+<body>
+    <header> 
+        <h1>Cristal Blanco</h1>
+            <div class="container">
+            <nav>
+                <a href="#">Home</a>
+                    
+                <a href="pages/portfolio.html">Portfolio</a>
+                    
+                <a href="pages/about.html">About</a>
+            </nav>
+        </div>
+    </header>
+    <main> 
+        <p> 
+        Hi, my name is Cristal & I am a student at Ada Developers Academy. 
+        This is my portfolio home screen! 
+        </p>
+    <div>
+        <img src="/images/il1d.png" alt="I love one direction" >
```

Comment:
The absolute paths on this page (and the pages under the pages folder) work under the VSCode Live Server preview, but that assumes that the site "lives" at the root of a web server. This is often true, but not always. It can be safer to use relative paths (paths not starting with `/`) to specify resource paths. The main down side to doing so is if you have pages that live in different folders (like index and the other pages) the paths to shared resources differ between pages. However, using relative paths can make any eventual deployment a little easier.